hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaConfiguration.java

```diff
@@ -209,15 +212,23 @@ private KafkaConsumerPool kafkaConsumersPool(KafkaProperties kafkaProperties, Br
                 kafkaProperties.getKafkaConsumer().getFetchMaxWaitMillis(),
                 kafkaProperties.getKafkaConsumer().getFetchMinBytes(),
                 kafkaProperties.getKafkaConsumer().getNamePrefix(),
-                kafkaProperties.getKafkaConsumer().getConsumerGroupName());
+                kafkaProperties.getKafkaConsumer().getConsumerGroupName(),
+                kafkaProperties.isSaslEnabled(),
+                kafkaProperties.getSaslJaasConfig());
 
         return new KafkaConsumerPool(config, brokerStorage);
     }
 
     private AdminClient brokerAdminClient(KafkaProperties kafkaProperties) {
         Properties props = new Properties();
-        props.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, kafkaProperties.getBootstrapKafkaServer());
-        props.put(CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG, kafkaProperties.getKafkaServerRequestTimeoutMillis());
+        props.put(BOOTSTRAP_SERVERS_CONFIG, kafkaProperties.getBootstrapKafkaServer());
+        props.put(SECURITY_PROTOCOL_CONFIG, DEFAULT_SECURITY_PROTOCOL);
+        props.put(REQUEST_TIMEOUT_MS_CONFIG, kafkaProperties.getKafkaServerRequestTimeoutMillis());
+        if (kafkaProperties.isSaslEnabled()) {
+            props.put("sasl.mechanism", "PLAIN");
+            props.put("security.protocol", "SASL_PLAINTEXT");
```

Comment:
In Frontend & Consumers those two above parameters can be configured but here are hardcoded.